dashboard/modules/job/cli.py

```diff
@@ -270,18 +278,29 @@ def submit(
         runtime_env_json=runtime_env_json,
         working_dir=working_dir,
     )
-    job_id = client.submit_job(
-        entrypoint=list2cmdline(entrypoint),
-        submission_id=submission_id,
-        runtime_env=final_runtime_env,
-        metadata=metadata_json,
-        entrypoint_num_cpus=entrypoint_num_cpus,
-        entrypoint_num_gpus=entrypoint_num_gpus,
-        entrypoint_memory=entrypoint_memory,
-        entrypoint_resources=entrypoint_resources,
-    )
 
-    _log_big_success_msg(f"Job '{job_id}' submitted successfully")
+    try:
+        job_id = client.submit_job(
+            entrypoint=list2cmdline(entrypoint),
+            submission_id=submission_id,
+            runtime_env=final_runtime_env,
+            metadata=metadata_json,
+            entrypoint_num_cpus=entrypoint_num_cpus,
+            entrypoint_num_gpus=entrypoint_num_gpus,
+            entrypoint_memory=entrypoint_memory,
+            entrypoint_resources=entrypoint_resources,
+        )
+
+        _log_big_success_msg(f"Job '{job_id}' submitted successfully")
+    except Exception as e:
+        if (
+            allow_exists
```

Comment:
Maybe we can also use an environment variable to control `allow_exists` so that we don't need to check the version on the KubeRay side.

Reply:
Added an environment varibale `RAY_JOB_SUBMISSION_ALLOW_EXISTS`